CovertInfiltration/Src/CovertInfiltration/Classes/X2EventListener_Infiltration.uc

```diff
@@ -234,6 +235,35 @@ static protected function EventListenerReturn ShouldCleanupCovertAction(Object E
 	return ELR_NoInterrupt;
 }
 
+static protected function EventListenerReturn TriggerPrototypeAlert(Object EventData, Object EventSource, XComGameState GameState, Name EventID, Object CallbackData)
+{
+	local XComGameState_Tech TechState;
+	local X2ItemTemplateManager ItemTemplateManager;
+	local X2ItemTemplate ItemTemplate;
+	local array<name> ItemRewards;
+	local name ItemName;
+
+	ItemTemplateManager = class'X2ItemTemplateManager'.static.GetItemTemplateManager();
+	
+	TechState = XComGameState_Tech(EventData);
+
+	if(TechState == none) return ELR_NoInterrupt;
+
+	ItemRewards = TechState.GetMyTemplate().ItemRewards;
+	foreach ItemRewards(ItemName)
+	{
+		`LOG("Evaluating Item:" @ string(ItemName));
```

Comment:
From `README.md`:

> Use custom log macros - `CI_Log`, `CI_Warn` and `CI_Trace`. Any PR that uses `ScriptLog` (eg. `log("Something")`) __will not be merged__

Reply:
Don't really need that log anymore, was for debugging the issue with armor prototypes. I'll remove it